components/utils/escapeRegExp.js

```diff
@@ -0,0 +1,10 @@
+/**
```

Comment:
Can you add some unit test for this file?

Reply:
Actually, on that same subject, could you also update the story to include at least one example with `/`, please?

Reply:
Added tests and added name with "/"